src/controls/Inactivemouse.js

```diff
@@ -0,0 +1,29 @@
+function debounce(delay, fn) {
```

Comment:
This might look cryptic but it's a simple function.

It will execute `fn` if after the said delay, however if you execute it before the delay the timer will be reset.